.pipelines/templates/e2e-windows-template.yaml

```diff
@@ -14,6 +14,9 @@ parameters:
   - name: windowsNodeImageVersion
     type: string
     default: AKSWindows-2019-containerd-test-2023.02.07
+  - name: windowsGpuDriverSuffix
+    type: string
+    default:
```

Comment:
Is default empty?

Reply:
It seems like that this is required and we set it in e2e-windows.yaml so I think that we should not set a default value (empty).

Reply:
It is not required for non-gpu scenario to set a value in e2e-windows.yaml. We don't want to force non-gpu scenario to set a value for gpu related stuffs in e2e-windows.yaml so I set a default value of an empty string. The empty string will be passed down so that we don't append a suffix to those non-gpu scenarios artifacts. 

For example, line 71 in e2e-windows-template.yaml:
      `artifact: ${{ parameters.windowsImage }}${{ parameters.windowsGpuDriverSuffix }}-clusterConfig`